terralab/commands/pipeline_runs_commands.py

```diff
@@ -96,6 +96,11 @@ def details(job_id: str) -> None:
         job_id_uuid
     )
 
+    pipeline_response: PipelineWithDetails = pipelines_logic.get_pipeline_info(
```

Comment:
i wish we didnt have to make this extra call but i guess there really isnt a way around it to give the user all the inputs (user supplied or not) the pipeline run used